install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -80,6 +80,11 @@ $ ansible-playbook [-i /path/to/file] \
 . After the playbook completes successfully,
 xref:../install/running_install.adoc#advanced-verifying-the-installation[verify the installation].
 
+[WARNING]
```

Comment:
Wondering if this warning should go before the procedure? I guess it depends if the warning is for that step specifically, or the entire procedure.